Fedoraware/Fedoraware-TF2/src/Core/Core.cpp

```diff
@@ -32,7 +32,7 @@ void CCore::OnLoaded()
 {
 	LoadDefaultConfig();
 
-	I::Cvar->ConsoleColorPrintf(Vars::Menu::Colors::MenuAccent, "%s Loaded!\n", Vars::Menu::CheatName.c_str());
+	I::Cvar->ConsoleColorPrintf(Vars::Menu::Colors::MenuAccent.Value, "%s Loaded!\n", Vars::Menu::CheatName.Value.c_str());
```

Comment:
I wonder if it would look cool to override the dereference operator to retrieve the value :D